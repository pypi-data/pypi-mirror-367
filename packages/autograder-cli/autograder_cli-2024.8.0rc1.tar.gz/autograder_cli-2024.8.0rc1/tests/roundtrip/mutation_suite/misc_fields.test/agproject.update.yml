project:
  name: Test Project 0d6f2cfa7bc44e32ba21cbc8e02d40d0
  timezone: America/Chicago
  course:
    name: Test Course
    semester: Summer
    year: 2014
  settings:
  instructor_files:
  - local_path: instructor_file1.txt
  - local_path: instructor_file2.txt
  - local_path: instructor_file3.txt
  student_files:
  - test1.py
  - test2.py
  - pattern: test*.py
    max_num_matches: 5
  mutation_suites:
  - name: Suite
    instructor_files_needed:
    - instructor_file1.txt
    - instructor_file3.txt
    - instructor_file2.txt
    read_only_instructor_files: true
    student_files_needed:
    - test2.py
    - test1.py
    allow_network_access: false
    deferred: false
