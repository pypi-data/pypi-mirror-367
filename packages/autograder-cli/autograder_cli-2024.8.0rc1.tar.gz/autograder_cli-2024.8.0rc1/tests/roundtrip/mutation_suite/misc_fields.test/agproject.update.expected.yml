project:
  name: Test Project 0d6f2cfa7bc44e32ba21cbc8e02d40d0
  timezone: America/Chicago
  course:
    name: Test Course
    semester: Summer
    year: 2014
  settings: {}
  instructor_files:
  - local_path: instructor_file1.txt
  - local_path: instructor_file2.txt
  - local_path: instructor_file3.txt
  student_files:
  - test1.py
  - test2.py
  - pattern: test*.py
    max_num_matches: 5
  mutation_suites:
  - name: Suite
    instructor_files_needed:
    - instructor_file1.txt
    - instructor_file2.txt
    - instructor_file3.txt
    student_files_needed:
    - test1.py
    - test2.py
